code/lightna/magento-os-backend/Index/Category.php

```diff
@@ -46,6 +46,6 @@ protected function getScanSelect(): Select
 
     public function getBatchSelect(array $ids): Select
     {
-        return $this->categories->getListSelect(['image', 'description']);
+        return $this->categories->getListSelect(['image', 'description', 'meta_description']);
```

Comment:
@vesan-83 here?

Reply:
damn, I was sure all category attributes are loaded like for products. @websnap , could you help to make sure all attributes are loaded for categories in the index?

Reply:
I could roll back this change so merge it, wdyt? so once meta desc is there for cats, it will be shown

Reply:
we can merge this change, but it needsto be fixed later